test/framework/toolchain.py

```diff
@@ -339,6 +353,20 @@ def test_optarch_generic(self):
                         else:
                             self.assertFalse(generic_flags in tc.get_variable(var))
 
+    def test_optarch_aarch64_heuristic(self):
```

Comment:
for completeness sake: this problem was fixed in https://github.com/geimer/easybuild-framework/pull/12